scripts/measures/2022/import-cost-measures.js

```diff
@@ -68,7 +68,6 @@ function convertCostCsvsToMeasures(costCSVRows) {
     }
     _.each(COST_CSV_COLUMN_NAMES, (measureKeyName, columnName) => {
       if (measureKeyName === 'submissionMethods') {
-        console.log('** ' + row[columnName]);
```

Comment:
This log statements was probably added for testing years back, and was never removed. Since it's in a loop, it clogs up the terminal during builds.